# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/lvm2/lvm_conf.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Tobias Hintze <th@rocklinux.org>:
	small fix for LVM2:
	don't scan nbd's by default

--- ./doc/example.conf	2004-04-16 18:11:34.000000000 +0200
+++ ./doc/example.conf	2005-01-17 15:10:59.320475111 +0100
@@ -37,7 +37,7 @@
     # that the cache file gets regenerated (see below).
 
     # By default we accept every block device:
-    filter = [ "a/.*/" ]
+    filter = [ "r/nbd.*/", "a/.*/" ]
 
     # Exclude the cdrom drive
     # filter = [ "r|/dev/cdrom|" ]
